x-pack/solutions/security/plugins/security_solution/public/common/components/rule_actions_field/index.tsx

```diff
@@ -10,6 +10,7 @@ import { EuiSpacer, EuiCallOut, EuiText } from '@elastic/eui';
 import React, { useCallback, useEffect, useMemo, useState } from 'react';
 import deepMerge from 'deepmerge';
 import ReactMarkdown from 'react-markdown';
+// eslint-disable-next-line @kbn/eslint/module_migration
```

Comment:
Added this to avoid migrating `styled-components` and to keep code unchanged as part of this PR.